src/vm/value.c

```diff
@@ -577,6 +577,8 @@ bool compareStringLess(Value a, Value b) {
     for (i = 0; firstString->chars[i] != '\0' || secondString->chars[i] != '\0'; i++) {
         if (firstString->chars[i] < secondString->chars[i]) {
             return true;
+        } else if (firstString->chars[i] > secondString->chars[i]) {
```

Comment:
Not sure how specific you like things but I don't believe the `else` here and below are necessary.

Reply:
Yeah I guess you're right since we're returning early the else keyword is a bit redundant. Will change that